src/dispatch/case/models.py

```diff
@@ -180,6 +199,8 @@ class CaseCreate(CaseBase):
     case_severity: Optional[CaseSeverityRead]
     case_type: Optional[CaseTypeRead]
     project: Optional[ProjectRead]
+    reporter: Optional[ParticipantUpdate]
```

Comment:
The Pydantic models have changed. You may want to merge the changes in `master` and add reporter, observer, and participants to the CaseReadMinimal model.